### PR TITLE
Adapted to change in Vidhance API.

### PR DESCRIPTION
**Pull request by emilwestergren**
Adapts to https://github.com/vidhance/ooc-vidproc/pull/1170

@sebastianbaginski peer review

### CODE REVIEW

File: nexus6/VidhanceProcessor.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ VidhanceProcessor::VidhanceProcessor(const char* cameraId) :
 		vidhance_motion_settings motionSettings = vidhance_settings_getMotion(settings);
 		vidhance_motion_sensor_settings sensorSettings = vidhance_motion_settings_getSensor(motionSettings);
 		vidhance_motion_sensor_gyro gyroSensor = vidhance_motion_sensor_gyro_new(GyroReader::getAngularVelocity, GyroReader::getInstance()->getSamplePeriod());
-		vidhance_motion_sensor_rotation rotationSensor = vidhance_motion_sensor_rotation_new(gyroSensor);
+		vidhance_motion_sensor_rotation rotationSensor = vidhance_motion_sensor_rotation_new((vidhance_motion_sensor)gyroSensor);
```
Comment (sebastianbaginski) [inline nexus6/VidhanceProcessor.cpp]: Does it give you a warning now ?

Comment (emilwestergren) [inline nexus6/VidhanceProcessor.cpp]: No, it gives me an error.

Comment (sebastianbaginski) [inline nexus6/VidhanceProcessor.cpp]: Ah right, there is no information about inheritance in C structs. Maybe we could include this information, I have a little idea I need to test first.

Comment (sebastianbaginski) [inline nexus6/VidhanceProcessor.cpp]: Yes it works, I will write a comment in the issue.

**Pull request merged**